qt-bugs@ issue : 193866
Trolltech task ID : 193338
bugs.kde.org number : none
applied: yes
author: Montel Laurent <montel@kde.org>

During porting qt3to4 port QGroupAction to Q3GroupAction
but not QAction to Q3Action (which is logical)
But it crashs when it's not a q3action.
Patch from TT.

Index: src/qt3support/widgets/q3action.cpp
===================================================================
--- src/qt3support/widgets/q3action.cpp	(revision 771122)
+++ src/qt3support/widgets/q3action.cpp	(working copy)
@@ -1664,7 +1664,7 @@
         if (d->dropdown) {
             if (!d->exclusive) {
                 QList<Q3Action*>::Iterator it(d->actions.begin());
-                if (!(*it))
+                if (it == d->actions.end() || !(*it))
                     return true;
 
                 Q3Action *defAction = *it;
